bash_it.sh

```diff
@@ -91,3 +91,5 @@ if [ -e "$HOME/.jekyllconfig" ]
 then
   . "$HOME/.jekyllconfig"
 fi
+
+set +T
```

Comment:
Can you please add a comment above this line to explain what it does and why it's needed? Ideally with a link back to this PR and your example/explanation. This would make it easier to understand and to maintain.

I'm afraid that without a comment, others will soon ask why this code is needed...